src/main/java/discord4j/discordjson/json/SearchGuildMembersRequest.java

```diff
@@ -0,0 +1,23 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableSearchGuildMembersRequest.class)
+@JsonDeserialize(as = ImmutableSearchGuildMembersRequest.class)
+public interface SearchGuildMembersRequest {
+
+    static ImmutableSearchGuildMembersRequest.Builder builder() {
+        return ImmutableSearchGuildMembersRequest.builder();
+    }
+
+    String query();
+
+    Possible<Optional<Integer>> limit();
```

Comment:
Not nullable

Reply:
`All parameters to this endpoint except for `query` are optional`

Reply:
Fixed